MergeHRISClient/api_client.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, configuration=None, header_name=None, header_value=None,
             self.default_headers[header_name] = header_value
         self.cookie = cookie
         # Set default User-Agent.
-        self.user_agent = 'OpenAPI-Generator/1.0.0/python'
+        self.user_agent = 'OpenAPI-Generator/1.0.1/python'
```

Comment:
This version almost certainly comes from the config, which means it should either be `1.3.1` or `1.2.2` now